### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Indominous**
fixed a few spelling mistakes, made it clearer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,38 +100,38 @@ To check that they're all correct, you can run the below command:
...
-ATLauncher supports custom themes. The process is fairly straight forward, but may require a lot of trial and error.
+ATLauncher supports custom themes. The process is fairly straight forward but may require a lot of trial and error.
 
 First you must create a `MyThemeName.java` in the `src/main/java/com/atlauncher/themes/` directory. Your theme should
 extend one of the base ATLauncher themes depending on what you need:
 
 -   `Dark` is the default theme and is a dark theme. It's a good place to start with some defaults for new dark themes.
 -   `Light` is a light theme. It's a good place to start with some defaults for new light themes.
 -   `ATLauncherLaf` is a base class which every theme MUST at some point extend. It provides some defaults including our
-    brand colours and some defaults. This shouldn't be extended from unless you need absolute power.
```
Comment (RyanTheAllmighty) [inline README.md:116]: This is not incorrect, launchers base language including all documentation is en-gb not en-us

**Review by RyanTheAllmighty — CHANGES_REQUESTED** [review]
One change and a change log entry must be added for any PR to be merged

**Comment by some-du6e** [comment]
["Running In Test"](https://github.com/ATLauncher/ATLauncher#running-in-test) really?

**Comment by RyanTheAllmighty** [comment]
Closing due to inactivity

**Pull request closed without merging**